wiki/People/The_Team/de.md

```diff
@@ -1,56 +1,103 @@
-| [Qualität Bestätigungs Team](/wiki/Modding/Quality_Assurance_Team) | Wir helfen dir Freude an osu! zu haben, beginnend mit der Sicherung der Qualität von Beatmaps |
+| ![][flag_CA] [awp](https://osu.ppy.sh/users/2650) | Allgemeiner Hausmeister und Helfer, Chibi-Künstler. [Barrel Roll Weekly](http://brw.twinkfish.com/) |
+| ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | Audio-Designer, [offizieller osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | Organisator und Administrator des World Cups |
+| ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | Allgemeiner Hausmeister, Community Manager |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | osu!-Entwickler, Entwickler von IRC-Integration für In-Game-Chat, Wartung der Website. [Blog](http://blog.echo.sh/) |
+| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour Gastgeber, World Cup Streamer, Wettbewerbsorganisator |
+| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | osu! Entwickler, Entwickler von osz2 und p2p Backend |
+| ![][flag_US] Jim | Ursprünglicher Website-Designer, Hosting-Provider in den Anfangstagen. [Brave New Games](http://www.bravegamer.com/) |
+| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | Organisator und Administrator des World Cups |
+| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | Organisator und Administrator des World Cups, QAT-Leiter, Community-Manager |
+| ![][flag_US] [LuigiHann](https://osu.ppy.sh/users/1079) | Epischer Skin-Designer, Mitwirkender am Default-Skin und Iconset. [DeviantArt](https://luigihann.deviantart.com/) |
+| ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | Berater für die Modding- und Mapping-Szene |
+| ![][flag_CA] [mm201](https://osu.ppy.sh/users/30655) | osu!-Entwickler, Schöpfer von mm-Slidern |
+| ![][flag_US] [nuudles](https://osu.ppy.sh/users/21312) | osu! iPhone-Entwickler |
+| ![][flag_CA] [Nyquill](https://osu.ppy.sh/users/682935) | osu! Entwickler, Mappers' Guild Organisator |
+| ![][flag_AU] [phill](https://osu.ppy.sh/users/53) | Forum Designer |
+| ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202) | osu!painter, coole Sachen-Macher-Typ der Extraklasse |
+| ![][flag_US] [Sarumaru](https://osu.ppy.sh/users/9427) | Ursprüngliches Pippi-Konzept und Artwork Designer. [DeviantArt](https://sarumaru.deviantart.com/) |
+| ![][flag_GR] [Sinistro](https://osu.ppy.sh/users/5530) | Früher Community-Manager und -Leiter, Mitwirkung an Sitzungsprotokollen und FAQ |
+| ![][flag_DE] [Tom94](https://osu.ppy.sh/users/1857058) | osu! und pp-Entwickler |
+| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | Allgemeiner Hausmeister, Organisator und Administrator des World Cups, Wiki-Wartung |
+| ![][flag_CN] [woc2006](https://osu.ppy.sh/users/1105845) | osu! Entwickler, osu!mania Modus Entwickler |
```

Comment:
```suggestion
| ![][flag_CN] [woc2006](https://osu.ppy.sh/users/1105845) | osu!-Entwickler, Entwickler für den osu!mania-Modus |
```
I think `Modus` can be dropped here, even if it is not corresponding so much to the English text anymore. It's not that much important anyway, because there can't be that much people who look through every single line here in my opinion.